fluent/client/client_test.go

```diff
@@ -102,10 +106,15 @@ var _ = Describe("Client", func() {
 
 		It("Sends the message", func() {
 			go func() {
-				client.SendMessage(&msg)
+				err := client.SendMessage(&msg)
+				if err != nil {
+					// TODO: Insert better check here
```

Comment:
I am not thrilled by this, any other options?  Put a nolint here as well?

Reply:
See https://github.com/IBM/fluent-forward-go/blob/07884c31afcee878c4e4db037d6959b060058244/fluent/client/tcp_test.go#L50